projects/lib/src/tournament.cpp

```diff
@@ -250,6 +250,15 @@ void Tournament::setPgnCleanupEnabled(bool enabled)
 	m_pgnCleanup = enabled;
 }
 
+void Tournament::setEpdOutput(const QString& fileName)
+{
+	if (fileName != m_epdFile.fileName())
+	{
+		m_epdFile.close();
```

Comment:
Good point. I did not even check for leaks, just coded EPD alongside the PGN methods, so this has been out of my focus. Now open PGN and EPD-files are getting closed in the tournament destructor.